README.md

```diff
@@ -37,19 +37,21 @@ Welcome to our community! We welcome contributors from many skillsets. Here's ho
 
 ## Projects
 
-The following projects are actively being developed:
-
-| 100 days | Archiving | Data Together | Web Monitoring | Website |
-|---|---|---|---|---|
-| Website for EDGI 100 Days Report at [100days. envirodatagov.org](https://100days.envirodatagov.org/) [**github.com/../100days**](https://github.com/edgi-govdata-archiving/100days) | Set of tools, workflows, and documentation for grassroots, event-driven, archiving | Distributed model for holding copies of archived and preserved data [**github.com/ datatogether**](https://github.com/datatogether) | Interface and backend for reviewing different versions of web pages [**github.com/../web-monitoring**](https://github.com/edgi-govdata-archiving/web-monitoring) | Project management and design support for EDGI's website at [envirodatagov.org](https://envirodatagov.org/) [**github.com/../edgi-website**](https://github.com/edgi-govdata-archiving/edgi-website) |
-
-There are additional [`community`](https://github.com/edgi-govdata-archiving/overview/labels/community) and [`infrastructure`](https://github.com/edgi-govdata-archiving/overview/labels/infrastructure) projects that support internal EDGI and outreach processes:
-
-| Tool Name | Description |
-|-----------|-------------|
-| [**EDGI Hubot**](https://github.com/edgi-govdata-archiving/edgi-hubot) | Chat bot for EDGI Slack built on the Hubot framework  |
-| [**EDGI Scripts**](https://github.com/edgi-govdata-archiving/edgi-scripts) | Code scripts for running and maintaining our digital infrastructure |
-| [**Video Call Landing Page**](https://github.com/edgi-govdata-archiving/video-call-landing-page) | Landing page app with important info that participants can be sent through prior to joining a video call  <br />[http://edgi-video-call-landing-page.herokuapp.com/](http://edgi-video-call-landing-page.herokuapp.com/) |
+Here are some projects we're building and maintaining right now.
+
+**Want to get involved?** Check out the emoji column to see all the different types of contribution we need for these projects!
+
+| Project (Click through to repo) | Description | Contribution type most needed ([emoji key from All Contributors](https://github.com/kentcdodds/all-contributors#emoji-key))|
+| --- | --- | --- |
+| [Web Monitoring](https://github.com/edgi-govdata-archiving/web-monitoring) | Tools around monitoring changes to government websites | 📖  🐛 💻
```

Comment:
Over in Web Monitoring, we take a lot of care to make sure these icons always have tooltips/title text, because I honestly find them super opaque (*especially* if you haven’t seen them before). For example: https://github.com/edgi-govdata-archiving/web-monitoring-db/blame/master/README.md#L223

Unrelatedly, I love the idea of having this info surfaced here, but also worry that we will do a very poor job keeping it accurate. Is this the best place for it?

Reply:
Good idea on tooltip text! Didn't know you could do that in Markdown. Will add.

The question on keeping it updated is important and deserves further discussion. One pipe-dream version of this would be issues tagged with skillsets auto-updating into a table like this one...

At the moment, my current thought is that it would be an improvement to have these here even if it's not totally up to date because it (1) highlights need for non-code contributions and (2) is supposed to get checked/updated by the PM (aka me) on a regular basis. So, I think it's a good starting point to get contributors into spaces and asking the right questions.

Reply:
Sounds reasonable to me 👍